### DISCUSSION TITLE
loading and loaded events

### DISCUSSION THREAD

**Discussion by oshumelka** [fancyapps/ui#452] [Q&A]
Hi! Could you tell please how to hook on to "loading" and "loaded" events? My [jsfiddle](https://jsfiddle.net/shumilka/68e0jxh9/ ) where I can't fire these events

**Comment by fancyapps** [comment]
Hi,

Inline content is not loading as it already exists on the page. Typically, you need `reveal` or `done` callbacks.

**Reply by oshumelka** [reply]
Thank you! 
So there are no beforeLoad/afterLoad (Fancybox v3) events for inline content now?

**Reply by fancyapps** [reply]
Hi,

You can not compare v3 and v5, because they work completely different. V5 (and v4) is built around Carousel component, therefore you can use events like `Carousel.createSlide` or `Carousel.removeSlide`. And load events occur only when something is loading. Inline content is just places inside the Carousel slide when that slide is created, and placed back when slide is removed.
